src/GimmeBundle/Resources/assets/gulpfile.js

```diff
@@ -450,4 +465,5 @@ gulp.task('default', [
     'mcc-basic-libs.js',
     'provabrasil.js',
     'jquery.libs.js',
+    'print.js'
```

Comment:
Next time `'print.js',` with comma in end of line is best. 👍

Reply:
Yes, sure! 👍